### DISCUSSION TITLE
__init__() got an unexpected keyword argument 'max_frame_length'

### DISCUSSION THREAD

**Discussion by rm2631** [miguelgrinberg/python-engineio#320] [General]
Release 4.5.0 introduced a bug for the following config on a gunicorn server : 

Flask==2.2.2
Flask-Cors==3.0.10
Flask-SocketIO==5.3.4
eventlet==0.30.2
python-socketio==5.7.2

Simply using python-engineio==4.4.1 resolves the issue.

**Comment by miguelgrinberg** [comment]
Yes, I can see the problem. You are using an eventlet release that is about 2 years old. I'll see what I can come up with to support old and new eventlets.

**Reply by miguelgrinberg** [reply]
Should be fixed in release 4.5.1. Please let me know if there are any more problems.

**Reply by rm2631** [reply]
Superb. Thank you. I recall that I use this version because of some incompatibility with either more recent versions of gunicorn or flask-socketio but I can't seem to find the reference right now. Guess I'll need to check again.
